runbotics/runbotics-orchestrator-ui/src/src-app/Actions/cloudExcel.actions.ts

```diff
@@ -690,7 +691,15 @@ const getCloudExcelActions: () => Record<string, IBpmnAction> = () => ({
         script: CloudExcelAction.CLOSE_SESSION,
         runner: Runner.DESKTOP_SCRIPT,
         form: {
-            schema: {},
+            schema: {
+                properties: {
+                    Input: {
+                        properties: {
+                            customCredentialId: propertyCustomCredential,
+                        }
+                    }
+                }
+            },
```

Comment:
should be more like:
`
            schema: {
                title: translate('Process.Details.Modeler.Actions.Common.Input'),
                type: 'object',
                properties: {
                    input: {
                        customCredentialId: propertyCustomCredential,
                    }
                }
            },
`

Reply:
slightly modified. Title will be empty because the section 'Input' is empty anyway